roles/cloud_devops_engineer.md

```diff
@@ -0,0 +1,91 @@
+# Cloud / DevOps Engineer
+
+Bristol, UK.
+
+Please apply for this role at [www.madetech.com/careers](https://www.madetech.com/careers).
+
+Our Senior Cloud Engineers deliver digital, data and technology outcomes that improve society. They do this by architecting and delivering cloud solutions, and coaching others to do so both within Made Tech and in public sector organisations.
```

Comment:
Replace all "Senior Cloud Engineer" with Cloud Engineer

Reply:
Should it be "Our Cloud & DevOps Engineers" ?